src/App.css

```diff
@@ -203,6 +208,18 @@
 .flex-end{
     justify-content: flex-end;
 }
+.maple-flex-col-xy-center{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items:center;
```

Comment:
i think you had a utility class for the properties in line 212,214,215, so maybe you could reuse that and refactor the code later on, as i can see lots of redundant code in the css.

Reply:
Thank you, Sammer.
I will Refactor the CSS